XRTK-Core/Packages/com.xrtk.core/Editor/EditorActiveProfileChangeHandler.cs

```diff
@@ -18,7 +18,7 @@ static EditorActiveProfileChangeHandler()
 
         private static void EditorApplication_hierarchyChanged()
         {
-            if (!MixedRealityToolkit.Instance.IsNull())
+            if (!(MixedRealityToolkit.Instance.IsNull() || MixedRealityToolkit.Instance.ActiveProfile.IsNull()))
```

Comment:
@FejZa this makes the xrtk auto-configure

Reply:
Ok thanks. I'll check it but not sure how this can be the cause of it?? Weird world.

Reply:
See #760